evap/evaluation/models.py

```diff
@@ -1483,6 +1482,7 @@ def create_superuser(self, email, password, first_name=None, last_name=None):
         return user
 
 
+@total_ordering
```

Comment:
The ordering relation we use here is not a [total order](https://en.wikipedia.org/wiki/Total_order). We would have
```python3
>>> user1, user2 = UserProfile(), UserProfile()
>>> user1 < user2 
False
>>> user2 < user1
False
>>> user2 == user1
False
```
because django's default implementation of `__eq__()` uses the primary key to detect equality.

I think this inconsistency will cause bugs in the future and thus would vote strongly against using [`@total_ordering`](https://docs.python.org/3/library/functools.html#functools.total_ordering) here.

If we want to use comparison for sorting, IMO, we should _only_ implement `__lt__()`. However, I'd also vote against doing that, see discussion below.